scripts/website/build-examples.sh

```diff
@@ -7,11 +7,21 @@ set -e # exit when error
 rm -rf website/examples
 mkdir website/examples
 
-for example in examples/*; do
-  if [ -d "$example" ]; then
-    name=$(basename "$example")
-    echo "Building $name example..."
+build_example() {
+  name=$1
+  echo "Building $name example..."
 
-    (cd "$example" && yarn && yarn build && cp -r "dist/" "../../website/examples/$name/")
-  fi
-done
+  (cd "$example" && yarn && yarn build && cp -r "dist/" "../../website/examples/$name/")
```

Comment:
does this work? `$example` is not defined here I think.

Reply:
you're right, I forgot to copy the right version: 24a4f68e